CHANGELOG.md

```diff
@@ -1,16 +1,8 @@
 # nf-core/oncoanalyser: Changelog
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project mostly adheres to
```

Comment:
"mostly"?

Reply:
As far as I have seen/understood most nf-core pipelines don't strictly adhere to semver, particularly around major versioning wrt breaking changes. See this thread for example: https://nfcore.slack.com/archives/CE56GDKN0/p1716478847517669

The same approach will likely be used here and the above being open about that.